server/handlers.go

```diff
@@ -184,38 +181,25 @@ func cancelProductHandler(c *gin.Context, client *mongo.Client) {
 	claims, _ := checkLogin(c)
 	userID := claims.Username
 
-	if data.Cancel1(client, string(userID), string(id)) != nil {
+	if data.Cancel1(client, string(userID), string(id)) != nil { // TODO: rename function here
```

Comment:
needs to change this to data.CancelOrder() functions including whoCancelled bool argument